awx/main/isolated/manager.py

```diff
@@ -208,7 +209,6 @@ def check(self, interval=None):
         status = 'failed'
         rc = None
         last_check = time.time()
-        dispatcher = CallbackQueueDispatcher()
```

Comment:
By creating this manually in the isolated code and calling `dispatcher.dispatch()`, we're missing _all_ of the special logic in `BaseTask.event_handler` (like the `host_name` association).